pkg/garbagecollection/gc.go

```diff
@@ -106,6 +106,7 @@ func (gc *GC) collectPods(option k8sapi.ListOptions, runningSet map[types.UID]bo
 			if err != nil {
 				return err
 			}
+			gc.logger.Infof("garbage collect pod (%v)", p.GetName())
```

Comment:
i just found we passed in the logger from controller. we should not do that. logger should have the correct component set to gc. we should create a logger inside this pkg.